docs/smart-contracts/quickstart.md

```diff
@@ -515,6 +614,27 @@ Having our account created, we can now deploy the contract:
       </TabItem>
     </Tabs>
   </TabItem>
+    <TabItem value="go" label="🐹 GO">
+    <Tabs groupId="cli-tabs">
+      <TabItem value="near-go-cli" label="Near-Go-Cli">
+        ```bash
```

Comment:
Same here, can you add a note that these commands need to be executed under `hello-world/contract` ?